src/pykeen/triples/utils.py

```diff
@@ -71,18 +71,18 @@ def load_triples(
 
     if encoding is None:
         encoding = "utf-8"
-
-    rv = np.loadtxt(
-        fname=path,
-        dtype=str,
-        comments="@Comment@ Head Relation Tail",
-        delimiter=delimiter,
-        encoding=encoding,
-    )
     if column_remapping is not None:
         if len(column_remapping) != 3:
             raise ValueError("remapping must have length of three")
-        rv = rv[:, column_remapping]
+    rv = pandas.read_csv(
+        path,
+        sep=delimiter,
+        encoding=encoding,
+        dtype=str,
+        header=None,
+        usecols=column_remapping,
+    ).values
```

Comment:
the pandas documentation at https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.values.html warns we should use `df.to_numpy()` instead of `df.values`. Any thoughts on this?

Reply:
then we better use `df.to_numoy()` :sweat_smile:

Reply:
273ac05 - I also got rid of the unused variable assignment